card/src/test/java/com/adyen/checkout/card/CardComponentParamsMapperTest.kt

```diff
@@ -122,6 +123,21 @@ internal class CardComponentParamsMapperTest {
         assertEquals(expected, params)
     }
 
+    @Test
+    fun `when there are any restricted card brand in payment method,they are removed from the params`() {
```

Comment:
@onurkaragunlu you can now add the `Code is unit tested` checkbox to your checklist 😆